### PR TITLE
Add no-ssl variant of scenario 1a

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by jsuchome** [comment]
Didn't you want to remove the line? Now you still show example for 1 cluster...

**Comment by dirkmueller** [comment]
sorry, this is not the commit I wanted

File: scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+    elements:
+      ntp-server:
+      - crowbar.qa4.cloud.suse.de
+      ntp-client:
+      - "@@service1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      - "@@controller1@@"
+      - "@@compute1@@"
+      - "@@compute2@@"
```
Comment (vmoravec) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml]: Can we rely on default values for the barclamps above or are all the configs needed for some reason?

**Comment by toabctl** [comment]
+1 (assuming this was tested)

**Comment by vmoravec** [comment]
For no-ssl file I think the `api` attributes with https are redundant. I would also remove all attributes that have `default` value.

**Comment by dirkmueller** [comment]
@vmoravec done

**Comment by vmoravec** [comment]
+1

**Pull request merged**